.env

```diff
@@ -1,13 +1,11 @@
+# The earlier tags of kimai needed specific build instructions.  We don't auto build these.
 # Tags that need the legacy building, we don't build them anymore anyway
-LEGACY_TAGS="0.8 0.8.1 0.9 1.0 1.0.1"
+# LEGACY_TAGS="0.8 0.8.1 0.9 1.0 1.0.1"
+
 # Supported distros
 DISTROS="apache-debian fpm-alpine fpm-debian"
 
-# The earlier tags of kimai needed specific build instructions.  We don't auto build these.
-# NUMBER_OF_TAGS_TO_DISCARD=16
-# Computed values, shouldn't need cahnging
-# ALLTAGS=$(git ls-remote --tags --refs https://github.com/kevinpapst/kimai2.git | awk --field-separator="/" '{print $3}')
-# TAGS=$(echo $ALLTAGS | awk -v START=$NUMBER_OF_TAGS_TO_DISCARD '{for(i=START;i<=NF;++i)print $i}')
-# ^^^ that was all too clever, just build the last tag
-
-TAGS=$(git ls-remote --tags --refs https://github.com/kevinpapst/kimai2.git | awk --field-separator="/" '{print $3}'|sort|tail -n2)
+# This script used to auto build new tags but that lead to pushing broken builds.
+# Now we'll just list the tags to build
+# TAGS=$(git ls-remote --tags --refs https://github.com/kevinpapst/kimai2.git | awk --field-separator="/" '{print $3}'|sort|tail -n2)
+TAGS="1.4 1.4.1"
```

Comment:
why hardcoded?

Reply:
Because the auto build is wait built and pushed the broken 1.4(.1) tags.  Now we have separated push from build we could go back to auto builds but I wanted to get the 1.4s out the door.